apps/contract-sync-job/src/main.ts

```diff
@@ -149,9 +68,7 @@ const main = async () => {
         });
 
         const detailsUri = ethers.utils.toUtf8String(contr.detailsUri);
-        const ipfsBlob = await fetchIPFS(detailsUri);
-        const ipfsText = await ipfsBlob.text();
-        const contributionDetails = JSON.parse(ipfsText);
+        const contributionDetails = await fetchIPFS(detailsUri);
```

Comment:
Can't this error out and stop the job? We want to make sure this error can be handled gracefully and the job can continue processing. It might be good to catch the parse error here continue processing